core/src/main/java/org/stellar/anchor/event/KafkaEventService.java

```diff
@@ -49,7 +47,15 @@ public void publish(AnchorEvent event) {
       }
       ProducerRecord<String, AnchorEvent> record = new ProducerRecord<>(topic, event);
       record.headers().add(new RecordHeader("type", event.getType().getBytes()));
-      producer.send(record);
+
+      // If the queue is offline, throw an exception
+      try {
+        producer.send(record).get();
+      } catch (Exception ex) {
+        ex.printStackTrace();
```

Comment:
If we are throwing an exception that's wrapping the causing Exception, it is not necessary to print the stack trace.

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/247705aab4bb5e0d048fba7d6ae2d9c768103249.